test_trained_model.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jul 27 19:43:21 2021
+
+@author: 7ml
+"""
+import sys, os, json
+import shutil
+
+import torch
+from torch.utils.tensorboard import SummaryWriter
+from torch.optim.lr_scheduler import ReduceLROnPlateau
+
+from utils.utils import (
+    dataset_loading_and_splitting,
+    train_validate_test_normal,
+    test,
+    setup_ddp,
+)
+from utils.models_setup import generate_model, get_device
+from data_utils.dataset_descriptors import (
+    AtomFeatures,
+    Dataset,
+)
+
+
+def test_trained_model(config_file: str = None, chosen_model: torch.nn.Module = None):
+
+    if config_file is None:
+        raise RuntimeError("No configure file provided")
+
+    if chosen_model is None:
+        raise RuntimeError("No model type provided")
+
+    run_in_parallel, world_size, world_rank = setup_ddp()
+
+    config = {}
+    with open(config_file, "r") as f:
+        config = json.load(f)
+    predicted_value_option = {1: 1, 2: 32, 3: 32, 4: 33, 5: 33, 6: 65}
+    config["output_dim"] = predicted_value_option[config["predicted_value_option"]]
+
+    dataset_options = {
```

Comment:
This should be a separate function (but I know you would prefer to merge this and clean it up later)

Reply:
@streeve 

I can remove this now, but maybe Pei needs it for her additional multi-task uni tests. 
If we need to have a test_trained model, I would like to have one that we can use on every model possible. 
@pzhanggit what do you think?

Reply:
No, I wouldn't remove, just make a function like `get_dataset`

Reply:
Are you suggesting to infer the output dimension from dataset? That also works for me.  As long as we do not limit the number of nodes/atoms to 32...

Reply:
We can just fix later